ui/src/app/pages/analysis/notebook-redirect/component.ts

```diff
@@ -138,15 +137,8 @@ export class NotebookRedirectComponent implements OnInit, OnDestroy {
 
     let initializedProgress = false;
     this.loadingSub = serverConfigStore.asObservable()
-      .flatMap(({useBillingProjectBuffer}) => {
-        if (useBillingProjectBuffer) {
-          return this.clusterService.listClusters(this.wsNamespace);
-        }
-        return  userProfileStore.asObservable()
-          .flatMap((profileStore) => {
-            return this.clusterService.listClusters(
-              profileStore.profile.freeTierBillingProjectName);
-          });
+      .flatMap(({}) => {
```

Comment:
Is there a more appropriate construction to use when the function parameter list is empty, like `.then()` ?

Reply:
serverConfigStore is no longer necessary here:

```
this.loadingSub = this.clusterService.listClusters(...)
  .do((resp) => { ....)
```